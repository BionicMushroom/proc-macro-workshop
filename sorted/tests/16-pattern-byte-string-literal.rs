#[sorted::check]
fn main() {
    let x = b"abc abc";

    #[sorted]
    match x {
        b"def abc" => println!("def abc"),
        b"abc def" => println!("abc def"),
        _ => println!("other"),
    }
}
