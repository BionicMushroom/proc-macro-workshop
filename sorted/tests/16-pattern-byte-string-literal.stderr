error: `b"abc def"` should sort before `b"def abc"`
 --> tests/16-pattern-byte-string-literal.rs:8:9
  |
8 |         b"abc def" => println!("abc def"),
  |         ^^^^^^^^^^
